docs/core/versions/versioning-docker.md

```diff
@@ -0,0 +1,15 @@
+
+### Docker
```

Comment:
This section reads more like a spec for future development than a doc. It's mostly future tense and discusses possibilities.